### PR TITLE
Minor changes and fixes

### PR DESCRIPTION
**Pull request by sstok**
|Q            |A  |
|---          |---|
|Bug Fix?     |yes|
|New Feature? |no |
|BC Breaks?   |yes|
|Deprecations?|no |
|Fixed Tickets|   |
|License      |MIT|

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,8 +1,6 @@
-src/Rollerworks/Bundle/SearchBundle/Resources/doc/_themes
-src/Rollerworks/Bundle/SearchBundle/Resources/doc/_build
 /phpunit.xml
 /composer.lock
-/composer.phar
```
Comment (cordoval) [inline .gitignore]: why the / in front of /vendor/?

File: composer.json [hunk 2]
```diff
@@ -11,15 +11,14 @@
         }
     ],
     "require": {
-        "php": ">=5.3.3",
-        "rollerworks/search-symfony-di": "~1.0@dev,>1.0.0-beta2",
+        "php": ">=5.4",
```
Comment (cordoval) [inline composer.json]: :+1: just in case we use 5.6 upwards now on prod

**Comment by cordoval** [comment]
:+1:

**Comment by sstok** [comment]
```
1) Rollerworks\Bundle\SearchBundle\Tests\Unit\DependencyInjection\RollerworksSearchBundleExtensionTest::testFieldsSetsGetRegisteredFromMetadata

Trying to get property of non-object
```

Thats not good :|

**Comment by cordoval** [comment]
not good :soccer: indeed

**Pull request merged**